tests/test_nb.py

```diff
@@ -0,0 +1,103 @@
+import os
+import pathlib
+import subprocess
+import shutil
+from unittest import mock
+
+import pytest
+
+import jupyter_client.kernelspec
+
+import json
+import sys
+
+PROJECT_ROOT = pathlib.Path(__file__).parent.absolute().parent
+NB_FILE_STD = pathlib.Path(__file__).parent.absolute() / 'notebook-std.ipynb'
+NB_FILE_VIP = pathlib.Path(__file__).parent.absolute() / 'notebook-vip.ipynb'
+
+
+def test_nb_vip_no_venv(tmp_path):
+    from vip_ipykernel.kernelspec import install as install_vip
+    import vip_ipykernel.kernelspec
+
+    os.chdir(tmp_path)
+
+    shutil.copy(NB_FILE_STD, ".")
+
+    #  Install the custom kernel
+    with mock.patch.dict(os.environ, {"HOME": str(tmp_path)}):
+        defaults = list(vip_ipykernel.kernelspec.make_vip_ipkernel_cmd.__defaults__)
+        defaults[1] = str(sys.executable)  # Set the python executable path to python used for pytest
+        defaults = tuple(defaults)
+        with mock.patch.object(vip_ipykernel.kernelspec.make_vip_ipkernel_cmd, "__defaults__", defaults):
+            dest = install_vip(user=True)
+
+        with open(str(dest) + "/kernel.json") as f:
+            spec = json.load(f)
+
+        #  The kernel should be installed in tmp_path
+        assert str(tmp_path) in jupyter_client.kernelspec.find_kernel_specs()['python3']
+
+        #  The kernel should be running with the same python that is running the tests
+        assert sys.executable in spec["argv"][0]
```

Comment:
```suggestion
        assert spec["argv"][0] == sys.executable
```

Maybe?

Reply:
Ah yes I think this is in a different comment somewhere, here it uses `in` because depending on the system and how the paths are resolved `sys.executable` could be `python3` but `spec["argv"][0]` may be `python3.8`. For example:

```ipython
Python 3.8.6 (default, Sep 30 2020, 04:00:38) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.19.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import sys

In [2]: from pathlib import Path

In [3]: sys.executable
Out[3]: '/usr/bin/python3'

In [4]: Path(sys.executable).resolve()
Out[4]: PosixPath('/usr/bin/python3.8')
```

So it's possible for sys.executable to be `/usr/bin/python3` but for the (resolved) path in the spec to be `/usr/bin/python3.8`